### ISSUE TITLE
Conflict with carpet mod?

### ISSUE THREAD

**Issue by ComputerJock** [EngineHub/WorldEdit#1678]
**Versions**
WorldEdit version:   worldedit-fabric-mc1.16.3-7.2.2-dist.jar
/we version
WorldEdit version 7.2.2
https://github.com/EngineHub/WorldEdit/
Platforms:
* Fabric-Official(7.2.2+fc3c499)
Capabilities
GAME)HOOKS: Fabric-Official
CONFIGURATION: Fabric-Official
USER_COMMANDS: Fabric-Official
PERMISSIONS: Fabric-Official
WORLDEDIT_CUI: Fabric-Official
WORLD_EDITING: Fabric-Official

Platform version: There is no "/version" command...
I'm running MC 1.16.4 with OpenJDK 14.0.2

**Describe the bug**
I am using the following mods for 1.16.4 on win7x64:
  autoconfig1u-3.3.1.jar
  cloth-config-4.10.13-fabric.jar
  fabric-api-0.30.3+1.16.jar
  fabric-carpet-1.16.5-1.4.26+v210210.jar
  fabricmod_VoxelMap-1.10.15_for_1.16.5.jar
  Hwyla-Mod-Fabric-1.16.3.jar
  litematica-fabric-1.16.4-0.0.0-dev.20210215.172138.jar
  lithium-fabric-mc1.16.5-0.6.3.jar
  malilib-fabric-1.16.4-0.10.0-dev.21+arne.5.jar
  minihud-fabric-1.16.4-0.19.0-dev.20210215.151117.jar
  modmenu-1.16.7.jar
  NoFog-1.16.2b1-Fabric.jar
  phosphor-fabric-mc1.16.3-0.7.1+build.11.jar
  sodium-fabric-mc1.16.3-0.1.0.jar
  WI-Zoom-1.3-MC1.16.5.jar
  WorldEditCUI-1.16.5+02.jar

If I use
  worldedit-fabric-mc1.16.3-7.2.0-dist.jar
then worldedit works and the game starts.

If I use
  worldedit-fabric-mc1.16.3-7.2.2-dist.jar
then the game hangs at the point where it is loading the world and the percentage is stuck at 0%

If I remove carpet mod, then worldedit 7.2.2 works.

In conclusion it appears, at least for me, that carpet mod & world edit 7.2.2 have some sort of interaction that prevents Minecraft starting.

**Comment by wizjany** [comment]
see pinned issue. literally don't even need to search.
